cmd/atlas/internal/client/driver.go

```diff
@@ -0,0 +1,109 @@
+package client
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"ariga.io/atlas/sql/mysql"
+	"ariga.io/atlas/sql/postgres"
+	"ariga.io/atlas/sql/schema"
+	"ariga.io/atlas/sql/sqlite"
```

Comment:
I think it's best if the driver implementations aren't coupled to the package, instead, they should be registered at runtime using a pattern similar to the way `sql` drivers are. This way users of the package can use the package for other drivers without modifying this code.